flixel/text/FlxText.hx

```diff
@@ -625,6 +633,11 @@ class FlxText extends FlxSprite
 
 	function set_text(Text:String):String
 	{
+		if (!multiLine && (Text.contains("\n") || Text.contains("\r")))
+		{
+			Text = Text.split("\n").join("").split("\r").join("");
+		}
+
```

Comment:
I disagree with this implementation, this isn't how FlxInputText works and it's not how FlxBitmapText works.

plus this will cause weird behavior in the following:
```hx
text.multiline = false;
text.text = myMultilineString;
text.multiline = true;
```

Reply:
> I disagree with this implementation, this isn't how FlxInputText works and it's not how FlxBitmapText works.

Do you mean that this should be implemented in a better way or that the text shouldn't be modified at al?

Reply:
the latter

Reply:
Alright, in that case though, the only change that this variable does is whether or not new lines can be added by the user in the new FlxInputText. It doesn't stop the text field from having new lines set through text.

Reply:
This field simply sets the underlying text field's multiline field, how input text handles new lines from user input is an entirely separate issue that should be handled by those input handlers

Reply:
Okay, I understand now 👍